Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/DefaultToolFactory.java

```diff
@@ -69,13 +69,13 @@ public Tool createTool(ToolType toolType, ToolOptionsViewController toolOptionsV
 			case ERASER:
 				return new EraserTool(createBrushToolOptionsView(toolLayout), contextCallback, toolOptionsViewController, toolPaint, workspace, commandManager);
 			case LINE:
-				return new LineTool(createBrushToolOptionsView(toolLayout), contextCallback, toolOptionsViewController, toolPaint, workspace, commandManager);
+				return new LineTool(createBrushToolOptionsView(toolLayout), contextCallback, toolOptionsViewController, toolPaint, workspace, commandManager, null);
```

Comment:
Please explain

Reply:
found a better option, thanks for pointing out 😄